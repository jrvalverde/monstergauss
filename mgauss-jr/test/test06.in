*MOL GE=S, FO, BA=E6*
*PRT T=Y
METHANOL TIMING TEST, 6-31G*
 0 1
   2
C 2 H H H
O 1 H

