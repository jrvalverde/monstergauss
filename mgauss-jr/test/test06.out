 WELCOME TO U OF T MONSTERGAUSS                                                                         UNX VERSION: 13 AUG 1991
 ******************************
0CONTROL CARDS:

 *MOL GE=S, FO, BA=E6*                                                   
 *PRT T=Y                                                                

0SEQUENCE    GROUP     MG OPTIONS         GROUP OPTIONS (USER CONTROLLABLE)                  LINKS TO BE CALLED          JUMP

     1         2      000012100100 300000000000000000000000000000000000000000000000   11000000000000000000000000000000     0
     2         3      000012100100 604000010000303000000000000000000000000000000000   11000101000000000000000000000000     0
     3         4      000012100100 100000000000000010300000000000000000000000000000   01000000000000000000000000000000     0
     4         5      000012100100 410000000000000000000000000000000000000000000000   10000000000000000000000000000000     0
     5         6      000012100100 000000000000000000001000000000000000000000000000   10000000000000000000000000000000     0
     8         7      000012100100 000000000000000000000000000000000000000000000000   01110000001000000000000000000000     0
0OPTION NUMBERS:      000000000111 111111122222222223333333333444444444455555555556   66666666677777777778888888888999
                      123456789012 345678901234567890123456789012345678901234567890   12345678901234567890123456789012

0CONVERSION FACTORS:  1 BOHR =  .52917706 ANGSTROMS
0                     1 HARTREE/BOHR = 8.2388584 MDYNE

                      1 HARTREE/ANGSTROM**2 = 4.3598149 MDYNE/ANGSTROM

                      1 ELECTRON*BOHR = 2.5417655 DEBYE

                      1 HARTREE/PARTICLE = 627.50959 KCAL/MOL = 2625.5001 KJ/MOL = 27.211611 EV/PARTICLE = 219474.67 CM(-1)/PARTICLE

                      1 ATOMIC UNIT OF ELECTRIC FIELD = 5.1422506D+09 VOLTS/CM
1METHANOL TIMING TEST, 6-31G*                                            
 ------------------------------------------------------------------------
0CHARGE  =   0          MULTIPLICITY  =  1

0*** 11:33:32 (ELAPSED CPU TIME       .02 S)   ENTER LINK  201 ***
 STANDARD GEOMETRICAL MODEL ROUTINE

0 2 EXPLICITLY NUMBERED ATOMS

0CONNECTION TABLE (AS INPUT)

 C 2 H H H                                                               
 O 1 H                                                                   
1CONNECTION TABLE (INTERNAL FORM)

0 N   AN   FC   TYPE   NONE   A(A)     T A   T R   T W   T X   T Y   T Z

  1    6    0     0      4      2      - 2   - 3   - 4   - 5   
  2    8    0     0      4      1      - 1   - 6   
  3    1    0     0      1      1      - 1   
  4    1    0     0      1      1      - 1   
  5    1    0     0      1      1      - 1   
  6    1    0     0      1      2      - 2   
0LEGEND: AN IS THE ATOMIC NUMBER, FC IS THE FORMAL CHARGE, AND T IS THE BOND TYPE TO THE ATOMS A TO Z
 ATOM TYPE: 0 SATURATED, 1 AROMATIC/DELOCALIZED, 2 DOUBLE BONDED, 22 PAIR OF DOUBLE BONDS, 3 TRIPLE BONDED

1
0*** 11:33:32 (ELAPSED CPU TIME       .00 S)   ENTER LINK  202 ***
0---------------------------------------------------------------------------------------------------
                                             Z MATRIX
0  I   AN   Z1      BL                  Z2     ALPHA                Z3      BETA                Z4
 ---------------------------------------------------------------------------------------------------
   1    6
   2    8    1   1.4300000   (  1)   
   3    1    1   1.0800000   (  2)       2   109.47122   (  6)   
   4    1    1   1.0800000   (  3)       2   109.47122   (  7)       3   120.00000   ( 10)       0
   5    1    1   1.0800000   (  4)       2   109.47122   (  8)       3   240.00000   ( 11)       0
   6    1    2    .9600000   (  5)       1   109.47122   (  9)       3      .00000   ( 12)       0
 ---------------------------------------------------------------------------------------------------


 ------------------------------------------------------------------------------------------------------------------------------------
                    COORDINATES IN ANGSTROMS                               COORDINATES IN BOHR                    ATOMIC ISOTOPE/MASS
   I   AN        X             Y             Z                        X             Y             Z               MASS#    MASS (AMU)
 ------------------------------------------------------------------------------------------------------------------------------------
   1    6     .0000000      .0000000      .0000000                 .0000000      .0000000      .0000000             12    12.00000000
   2    8     .0000000      .0000000     1.4300000                 .0000000      .0000000     2.7023091             16    15.99491464
   3    1    1.0182338      .0000000     -.3600000                1.9241835      .0000000     -.6803016              1     1.00782504
   4    1    -.5091169     -.8818163     -.3600000                -.9620917    -1.6663918     -.6803016              1     1.00782504
   5    1    -.5091169      .8818163     -.3600000                -.9620917     1.6663918     -.6803016              1     1.00782504
   6    1     .9050967      .0000000     1.7500000                1.7103853      .0000000     3.3070217              1     1.00782504
 ------------------------------------------------------------------------------------------------------------------------------------
0*** 11:33:32 (ELAPSED CPU TIME       .00 S)   ENTER LINK  301 ***

0PROJECTING FROM STO-3G BASIS (5D/7F)

0BASIS SET TYPE: STANDARD 6-31G*      ( 6D/10F)
 POLARIZATION  : D ON LI-NE   D ON NA-AR   D ON K-KR   D ON RB-XE                                     
 DIFFUSE FUNCS : NONE                                                                                 
 CONTAINS  16 SHELLS AND  38 BASIS FUNCTIONS
0*** 11:33:32 (ELAPSED CPU TIME       .02 S)   ENTER LINK  302 ***
0RYSSET ROOT STATISTICS:     INTERPOLATED:     1499     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:     3235
0*** 11:33:32 (ELAPSED CPU TIME       .08 S)   ENTER LINK  306 ***
0STANDARD CUTOFFS SELECTED IN SHELL.
0SHELL: FMTGEN WAS CALLED    18031 TIMES
0*** 11:33:33 (ELAPSED CPU TIME       .98 S)   ENTER LINK  308 ***
0RYSSET ROOT STATISTICS:     INTERPOLATED:    38501     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:    11470
 PHOENIX SHELL PASSES:     3751, NUMBER PROCESSED:     1496, FAST PASSES:     3751


     165183 TWO ELECTRON INTEGRALS WRITTEN OUT ON UNIT  9 WITH INTEGRAL CUTOFF  1.00000D-10
0*** 11:33:35 (ELAPSED CPU TIME      2.26 S)   ENTER LINK  402 ***

0PH INITIAL GUESS ... SCALE FACTOR (K/2) =  .8750
 SMALLEST EIGENVALUE OF OVERLAP IS  2.25795D-01
 SMALLEST EIGENVALUE OF OVERLAP IS  9.49695D-03
 COEFFICIENTS PROJECTED TO CURRENT BASIS
0*** 11:33:35 (ELAPSED CPU TIME       .12 S)   ENTER LINK  501 ***
1CLOSED SHELL SCF               NUCLEAR REPULSION ENERGY IS     40.314883514  HARTREES
0CONVERGENCE ON DENSITY MATRIX REQUIRED TO EXIT IS  1.0000D-09
0 CYCLE   ELECTRONIC ENERGY      TOTAL ENERGY    CONVERGENCE   EXTRAPOLATION

    1       -154.916879883      -114.601996370
    2       -155.273581444      -114.958697930   1.37795D-02              
    3       -155.323571658      -115.008688145   9.14382D-03              
    4       -155.338528627      -115.023645114   5.30427D-03              
    5       -155.343886276      -115.029002762   3.42890D-03      4-POINT 
    6       -155.347949455      -115.033065941
    7       -155.347168348      -115.032284834   5.00400D-05              
    8       -155.347169112      -115.032285598   2.27778D-05              
    9       -155.347169240      -115.032285726   9.92760D-06              
   10       -155.347169265      -115.032285751   5.52214D-06              
   11       -155.347169270      -115.032285757   2.81730D-06              
   12       -155.347169272      -115.032285758   1.76705D-06              
   13       -155.347169272      -115.032285759   1.01296D-06              
   14       -155.347169273      -115.032285759   6.48172D-07              
   15       -155.347169273      -115.032285759   3.90921D-07      4-POINT 
   16       -155.347169273      -115.032285759
   17       -155.347169273      -115.032285759   7.63939D-09              
   18       -155.347169273      -115.032285759   3.12272D-09              
   19       -155.347169273      -115.032285759   1.50409D-09              
   20       -155.347169273      -115.032285759   7.04709D-10              
0AT TERMINATION TOTAL ENERGY IS      -115.032286  HARTREES
0*** 11:33:43 (ELAPSED CPU TIME      7.09 S)   ENTER LINK  601 ***
1MOLECULAR ORBITALS                      9 OCCUPIED MO

                           1          2          3          4          5          6          7          8          9         10
 
    EIGENVALUES---   -20.55283  -11.26762   -1.34410    -.92705    -.68335    -.61715    -.59188    -.49700    -.44289     .21843
 
   1  1   C   1S        .00002     .99574    -.06859    -.17828    -.01761     .00000    -.00324    -.00959     .00000     .04682
   2  1   C   2S'      -.00024     .02758     .12590     .34660     .03018     .00000     .00527     .02481     .00000    -.02056
   3  1   C   2PX'      .00002     .00009     .00575    -.00659     .21222     .00000    -.28404    -.28154     .00000    -.09507
   4  1   C   2PY'      .00000     .00000     .00000     .00000     .00000    -.38888     .00000     .00000    -.23753     .00000
   5  1   C   2PZ'     -.00036     .00069     .09132    -.07117    -.18359     .00000    -.26954     .13678     .00000     .15513
   6  1   C   2S"      -.00002    -.01228     .04859     .32636     .06725     .00000     .05465    -.00395     .00000    -.95578
   7  1   C   2PX"     -.00025    -.00004     .00056     .00548     .08575     .00000    -.14441    -.12858     .00000    -.32916
   8  1   C   2PY"      .00000     .00000     .00000     .00000     .00000    -.18906     .00000     .00000    -.09639     .00000
   9  1   C   2PZ"      .00014     .00038     .00173    -.03813    -.05528     .00000    -.10427     .04264     .00000     .55153
  10  1   C   3DXX      .00003    -.00179    -.00424     .01330     .02376     .00000    -.00268    -.02316     .00000     .01462
  11  1   C   3DYY      .00007    -.00180    -.00531     .01264    -.00068     .00000     .03143     .01198     .00000     .00858
  12  1   C   3DZZ     -.00007    -.00170     .02375    -.00490    -.02158     .00000    -.02812     .01266     .00000    -.00191
  13  1   C   3DXY      .00000     .00000     .00000     .00000     .00000     .02314     .00000     .00000     .01891     .00000
  14  1   C   3DXZ     -.00002     .00001     .00183    -.00133     .00197     .00000     .01104     .02904     .00000    -.00264
  15  1   C   3DYZ      .00000     .00000     .00000     .00000     .00000     .00811     .00000     .00000     .03576     .00000
 
  16  2   O   1S       -.99465    -.00016    -.20308     .05960     .04141     .00000    -.02002     .04943     .00000     .07986
  17  2   O   2S'      -.02117     .00009     .45777    -.13413    -.09424     .00000     .04890    -.10918     .00000    -.06005
  18  2   O   2PX'     -.00109     .00004     .06827    -.07701     .38376     .00000    -.06060     .37690     .00000    -.17826
  19  2   O   2PY'      .00000     .00000     .00000     .00000     .00000    -.32466     .00000     .00000     .55463     .00000
  20  2   O   2PZ'      .00072    -.00012    -.04859    -.17105     .18540     .00000     .37804    -.24397     .00000    -.01494
  21  2   O   2S"      -.00396     .00155     .43149    -.18382    -.16715     .00000     .07474    -.22895     .00000   -1.08022
  22  2   O   2PX"      .00049    -.00010     .04022    -.04238     .23682     .00000    -.04167     .29270     .00000    -.41197
  23  2   O   2PY"      .00000     .00000     .00000     .00000     .00000    -.21537     .00000     .00000     .46296     .00000
  24  2   O   2PZ"     -.00033     .00009    -.02885    -.08835     .10916     .00000     .26073    -.19337     .00000    -.03668
  25  2   O   3DXX      .00407    -.00002     .02189    -.01687     .03742     .00000     .00836     .02533     .00000     .03023
  26  2   O   3DYY      .00386     .00000    -.00116    -.00050    -.00546     .00000     .00689    -.00797     .00000     .04824
  27  2   O   3DZZ      .00421    -.00083     .01636     .01747    -.01714     .00000    -.01741     .00755     .00000     .03991
  28  2   O   3DXY      .00000     .00000     .00000     .00000     .00000    -.01249     .00000     .00000     .02386     .00000
  29  2   O   3DXZ      .00006    -.00002     .00509    -.00977     .00053     .00000     .02471    -.00933     .00000     .00212
  30  2   O   3DYZ      .00000     .00000     .00000     .00000     .00000     .01362     .00000     .00000    -.00820     .00000
 
  31  3   H   1S'       .00007     .00005     .02688     .13860     .16684     .00000    -.10680    -.22294     .00000     .01603
  32  3   H   1S"       .00014     .00252     .00651     .03809     .11595     .00000    -.08873    -.21571     .00000     .93225
 
  33  4   H   1S'       .00000     .00007     .02434     .14030    -.00292     .18837     .14400     .07670     .16057    -.00497
  34  4   H   1S"       .00002     .00252     .00315     .03999    -.00812     .12937     .10428     .06675     .17362     .31552
 
  35  5   H   1S'       .00000     .00007     .02434     .14030    -.00292    -.18837     .14400     .07670    -.16057    -.00497
  36  5   H   1S"       .00002     .00252     .00315     .03999    -.00812    -.12937     .10428     .06675    -.17362     .31552
 
  37  6   H   1S'      -.00038    -.00002     .12321    -.10150     .21259     .00000     .06534     .13023     .00000     .06828
  38  6   H   1S"       .00021     .00022    -.00188    -.02338     .10357     .00000     .04845     .08659     .00000    1.22015
 

                          11         12         13         14         15         16         17         18         19         20
 
    EIGENVALUES---      .27793     .32640     .33113     .38584     .75919     .76236     .78559    1.08509    1.12892    1.14000
 
   1  1   C   1S        .13953    -.04085     .00000    -.05210    -.00260     .00000    -.03874     .00300     .05754     .00000
   2  1   C   2S'      -.11830     .02334     .00000     .04027     .01664     .00000     .41222     .06522     .53642     .00000
   3  1   C   2PX'      .07164     .30666     .00000    -.11009    -.71352     .00000     .04757    -.52522     .51650     .00000
   4  1   C   2PY'      .00000     .00000    -.34815     .00000     .00000    -.72927     .00000     .00000     .00000    -.88632
   5  1   C   2PZ'      .06525     .08934     .00000     .18797     .02723     .00000    1.00177     .07494     .01227     .00000
   6  1   C   2S"     -2.49959     .66408     .00000     .73955     .00975     .00000    -.40818    -.36699   -1.68613     .00000
   7  1   C   2PX"      .31410    1.34669     .00000    -.56831    1.61316     .00000    -.12018    1.07814    -.89709     .00000
   8  1   C   2PY"      .00000     .00000   -1.54048     .00000     .00000    1.68851     .00000     .00000     .00000    1.35570
   9  1   C   2PZ"      .23732     .62456     .00000    1.44342     .01323     .00000   -1.24543    -.09473     .17065     .00000
  10  1   C   3DXX      .00182    -.00157     .00000     .02021    -.08897     .00000     .03809     .08393     .03545     .00000
  11  1   C   3DYY      .00923     .01532     .00000     .01324     .09450     .00000     .02892    -.02598     .11852     .00000
  12  1   C   3DZZ      .01002    -.00806     .00000    -.01495    -.01073     .00000    -.05793    -.08958    -.05870     .00000
  13  1   C   3DXY      .00000     .00000    -.01456     .00000     .00000     .10515     .00000     .00000     .00000    -.15566
  14  1   C   3DXZ      .00237     .00190     .00000     .00308     .04040     .00000     .00329    -.15429     .06363     .00000
  15  1   C   3DYZ      .00000     .00000     .00146     .00000     .00000     .02787     .00000     .00000     .00000    -.14864
 
  16  2   O   1S       -.05068     .04688     .00000     .03789    -.00711     .00000     .01822     .02435     .01449     .00000
  17  2   O   2S'       .01088    -.03127     .00000    -.03078     .07392     .00000    -.23389    -.01134     .69050     .00000
  18  2   O   2PX'      .09195    -.07119     .00000     .15605    -.07332     .00000    -.01810    -.33507    -.41943     .00000
  19  2   O   2PY'      .00000     .00000     .08036     .00000     .00000    -.17085     .00000     .00000     .00000     .20456
  20  2   O   2PZ'     -.04334     .12304     .00000     .28765     .07427     .00000     .27249     .16090     .25326     .00000
  21  2   O   2S"       .78618    -.74887     .00000    -.61072    -.03797     .00000     .75011    -.02167   -1.14625     .00000
  22  2   O   2PX"      .19648    -.25303     .00000     .42938    -.15717     .00000     .04576    -.21334     .37704     .00000
  23  2   O   2PY"      .00000     .00000     .23809     .00000     .00000    -.21192     .00000     .00000     .00000    -.28098
  24  2   O   2PZ"     -.16564     .37185     .00000     .88321     .01794     .00000    -.04772    -.45067    -.49451     .00000
  25  2   O   3DXX     -.02787     .00780     .00000     .03309    -.01241     .00000    -.10998     .13317     .24790     .00000
  26  2   O   3DYY     -.03398     .03580     .00000     .01475     .04310     .00000    -.10840    -.05205     .19722     .00000
  27  2   O   3DZZ     -.03825     .01257     .00000    -.01271     .03257     .00000     .11864     .01481     .28545     .00000
  28  2   O   3DXY      .00000     .00000    -.00445     .00000     .00000     .01336     .00000     .00000     .00000     .01594
  29  2   O   3DXZ     -.00689    -.00340     .00000     .01045    -.00586     .00000     .01917     .14737    -.02695     .00000
  30  2   O   3DYZ      .00000     .00000     .00506     .00000     .00000     .01168     .00000     .00000     .00000     .10617
 
  31  3   H   1S'      -.02159    -.02078     .00000     .07357    -.49883     .00000    -.07885     .47730    -.03567     .00000
  32  3   H   1S"       .75239   -1.82979     .00000     .87742    -.29977     .00000    -.00740   -1.02323    1.05605     .00000
 
  33  4   H   1S'       .00610     .03672    -.03803     .04675     .25266     .42293    -.10644    -.08741     .48866    -.66290
  34  4   H   1S"      1.32583     .77210   -1.70100    -.23464     .15408     .32411    -.05473     .54053    -.14343    1.34757
 
  35  5   H   1S'       .00610     .03672     .03803     .04675     .25266    -.42293    -.10644    -.08741     .48866     .66290
  36  5   H   1S"      1.32583     .77210    1.70100    -.23464     .15408    -.32411    -.05473     .54053    -.14343   -1.34757
 
  37  6   H   1S'      -.05114    -.02520     .00000    -.00826    -.13884     .00000    -.06798     .88415     .04548     .00000
  38  6   H   1S"      -.66266     .22714     .00000   -1.08119    -.05238     .00000     .16534    -.45324     .32638     .00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS                  (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1          2          3          4          5          6
 
         1   C         4.78187     .21158     .39428     .39790     .39790    -.02537
 
         2   O          .21158    8.38176    -.03043    -.03935    -.03935     .26273
 
         3   H          .39428    -.03043     .57334    -.03340    -.03340    -.01037
 
         4   H          .39790    -.03935    -.03340     .54976    -.04114     .00316
 
         5   H          .39790    -.03935    -.03340    -.04114     .54976     .00316
 
         6   H         -.02537     .26273    -.01037     .00316     .00316     .32771
 
0TOTAL ATOMIC CHARGES AND NET ATOMIC CHARGES

                           1          2
 
         1   C         6.15816    -.15816
 
         2   O         8.74693    -.74693
 
         3   H          .86002     .13998
 
         4   H          .83693     .16307
 
         5   H          .83693     .16307
 
         6   H          .56102     .43898
 
1BOND ORDER AND VALENCE INDICES (I. MAYER, I.J.Q.C., 29, 477-483 (1986))


          BOND ORDER MATRIX

                           1          2          3          4          5          6
 
         1   C          .00000     .86865     .96404     .95504     .95504    -.00111
 
         2   O          .86865     .00000    -.00381    -.00569    -.00569     .77313
 
         3   H          .96404    -.00381     .00000    -.01198    -.01198    -.00251
 
         4   H          .95504    -.00569    -.01198     .00000    -.01240     .00205
 
         5   H          .95504    -.00569    -.01198    -.01240     .00000     .00205
 
         6   H         -.00111     .77313    -.00251     .00205     .00205     .00000
 
0                      VALENCE

                           1
 
         1   C         3.74166
 
         2   O         1.62660
 
         3   H          .93376
 
         4   H          .92702
 
         5   H          .92702
 
         6   H          .77361
 
0*** 11:33:43 (ELAPSED CPU TIME       .04 S)   ENTER LINK  702 ***
0RYSSET ROOT STATISTICS:     INTERPOLATED:     1567     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:     3167
0*** 11:33:43 (ELAPSED CPU TIME       .12 S)   ENTER LINK  703 ***
0DTWOL: FMTGEN WAS CALLED    18444 TIMES.
0*** 11:33:45 (ELAPSED CPU TIME      1.34 S)   ENTER LINK  704 ***
0RYSSET ROOT STATISTICS:     INTERPOLATED:    31442     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:     7976
 DPHNIX SHELL PASSES:     3683, FAST PASSES:     3683
0*** 11:33:49 (ELAPSED CPU TIME      3.11 S)   ENTER LINK  711 ***
1INTERNAL COORDINATE DERIVATIVES (MDYNE OR MDYNE-ANGSTROM/RADIAN)

0  I   AN   Z1     BL                  Z2      ALPHA                Z3      BETA                 Z4

   1    6
   2    8    1    .124911   (  1)   
   3    1    1   -.025994   (  2)       2     -.049778   (  6)   
   4    1    1   -.016204   (  3)       2     -.002696   (  7)       3      .000977   ( 10)       0
   5    1    1   -.016204   (  4)       2     -.002696   (  8)       3     -.000977   ( 11)       0
   6    1    2    .143967   (  5)       1      .001621   (  9)       3      .000000   ( 12)       0
0GRADIENT LENGTH OVER ALL INTERNAL COORDINATES (NO CONSTRAINTS):  5.77545D-02
 LARGEST (ABSOLUTE) GRADIENT COMPONENT:      .143967

0*** 11:33:49 (ELAPSED CPU TIME       .03 S)   ENTER LINK  101 ***

0NORMAL TERMINATION, TOTAL CPU TIME FOR THIS CALCULATION WAS      .254 MINUTES.
13.9u 1.4s 0:20 76%
